### PR TITLE
Add p2p-circuit multiaddr

### PR DESCRIPTION
**Pull request by vyzo**

### CODE REVIEW

File: table.csv [hunk 1]
```diff
@@ -162,6 +162,7 @@ http,               ,                         0x01E0
 https,              ,                         0x01BB
 ws,                 ,                         0x01DD
 onion,              ,                         0x01BC
+p2p-circuit,        ,                         0x0122
```
Comment (Stebalien) [inline table.csv]: These should be sorted.

Comment (Stebalien) [inline table.csv]: They aren't ... :cry:

Comment (vyzo) [inline table.csv]: I can sort.

Comment (vyzo) [inline table.csv]: Not sure we want to sort though :)

Comment (Stebalien) [inline table.csv]: Yeah, just leave them as-is for now. We need to clean this all up in one go.

Comment (lgierth) [inline table.csv]: They're sorted by topic :)

Comment (vyzo) [inline table.csv]: so shall we merge?  I don't have perms on mulitformats.

**Pull request merged**